src/main/cc/proxy_client/proxy_client.cc

```diff
@@ -328,6 +383,9 @@ int ComputeInputs(int argc, char** argv, const char** env, const string& cwd, co
     inputs->insert("prebuilts/jdk/jdk9/linux-x86");
     inputs->insert("external/icu");
     FindAllFilesFromCommand(argc, argv, inputs);
+  } else if (getenv("RUN_ALL_REMOTELY")) {
+    use_args_inputs = true;
+    FindAllFilesFromCommand(argc, argv, inputs);
```

Comment:
We really need to clean up this entire function. Not in this CL, but in general. Lines 312-330 already try to figure out the inputs from command, but badly, and now you're doing it here better. We need to consolidate these.

Reply:
+1. Added to backlog (b/131301437).